src/couchapps/ReqMgr/updates/updaterequest.js

```diff
@@ -36,6 +36,19 @@ function(doc, req) {
     	return true;
 	}
 	
+	function updateTaskStepChain(chainType, prop, value) {
+		if (doc[chainType + "Chain"]) {
+			var numChain = doc[chainType + "Chain"];
```

Comment:
Don't we have to first test whether this `doc` contains this key (TaskChain or StepChain)? If not, just bail out.